examples/tutorials/core_api_mnist/model_def_checkpoints.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import print_function
```

Comment:
`# Save and load to and from checkpoint storage.' 
'# Introduce preemption support.`